snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java

```diff
@@ -38,6 +50,36 @@ public static void setup() {
     public static void shutdown() {
         PrimitiveData.stop();
     }
+    @Test
+    public void testDescriptionSemantics()throws IOException {
+        String sourceFilePath ="C:\\Users\\patrichards\\Solor\\SnomedCT_InternationalRF2_PRODUCTION_20240201T120000Z\\Full\\Terminology\\sct2_Description_Full-en_INT_20240201.txt";
+
+        try(BufferedReader br = new BufferedReader(new FileReader(sourceFilePath))){
+            String line;
+            while((line = br.readLine()) != null){
+                if (line.startsWith("id")) continue;
+                String[] columns = line.split("\\t");
+
+                //pass these args in assertion method
+                long effectiveTimeToLong=dateStringToEpochMillis(columns[1]);
+                EntityProxy.Concept descriptionStatus = Integer.parseInt(columns[2]) == 1 ? TinkarTerm.ACTIVE_STATE : TinkarTerm.INACTIVE_STATE;
```

Comment:
This was my bad. After talking to Jonathan we're going to go with either StateSet.ACTIVE or StateSet.INACTIVE.